tasks/43_brackets-indexes/brackets-indexes.js

```diff
@@ -0,0 +1,24 @@
+export const findCloseIndex = (str, num) => {
```

Comment:
for this question better approach will be using stack instead of counters. Try solve with stack. Also what happen when you will end up with something like this: {()({)}} ? it's follow up but IMO it's important :) take into consideration